packages/velo-external-db/test/resources/e2e_resources.js

```diff
@@ -55,23 +55,32 @@ const firestoreTestEnvInit = async() => await dbInit(firestore)
 const mssqlTestEnvInit = async() => await dbInit(mssql)
 const mongoTestEnvInit = async() => await dbInit(mongo)
 const googleSheetTestEnvInit = async() => await dbInit(googleSheet)
-const airtableTestEnvInit = async() => await dbInit(airtable)
+const airTableTestEnvInit = async() => await dbInit(airtable)
 const dynamoTestEnvInit = async() => await dbInit(dynamo)
 const bigqueryTestEnvInit = async() => await dbInit(bigquery)
 
 
-const testSuits = () => [
-    ['MySql', mysqlTestEnvInit],
-    ['Postgres', postgresTestEnvInit],
-    ['Spanner', spannerTestEnvInit],
-    ['Firestore', firestoreTestEnvInit],
-    ['Sql Server', mssqlTestEnvInit],
-    ['Mongo', mongoTestEnvInit],
-    ['Google-sheet', googleSheetTestEnvInit],
-    ['Airtable', airtableTestEnvInit],
-    ['DynamoDb', dynamoTestEnvInit],
-    ['BigQuery', bigqueryTestEnvInit]
-].filter( ([name]) => name.toLowerCase() === process.env.TEST_ENGINE || (name === 'Sql Server' && process.env.TEST_ENGINE === 'mssql') )
+const testSuits = {
+    mysql: ['MySql', mysqlTestEnvInit],
```

Comment:
lets convert the array to an object, { name: 'MySql', target: 'mysql',  setup: mysqlTestEnvInit }
this way we can have more clarity on which is what are we using and comparing, WDYT ?

Reply:
Agree